Android.gitignore

```diff
@@ -30,3 +30,7 @@ proguard/
 
 # Android Studio captures folder
 captures/
+
+# Idea
+.idea/
+*.iml
```

Comment:
This rule is new to me - should we add that here? https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore